web/models/pair.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Pairmotron.Pair do
+  use Pairmotron.Web, :model
+
+  schema "pairs" do
+    field :year, :integer
+    field :week, :integer
+    field :pair_group, :integer
+    belongs_to :user, Pairmotron.User
```

Comment:
Why did you choose to have one pair per user rather than a pair with a has_many relationship to users? That way if you had 10 users, you would have 5 pairs (and 5 db rows).